rest-service/manager_rest/rest/resources_v2.py

```diff
@@ -761,6 +757,18 @@ def get(self, _include=None, filters=None,
         :rtype: :class:`manager_rest.storage.storage_manager.ListResult`
 
         """
+        if 'execution_id' not in filters:
```

Comment:
Yes, I agree. The goal in this PR was to support what the client needs and then implement missing features as required.